tests/Resources/Page/ODH/AiApps/ManagedStarburst.resource

```diff
@@ -0,0 +1,105 @@
+*** Settings ***
+Documentation    Collections of keywords to interact with Managed Starburst
+Library    OperatingSystem
+Library    OpenShiftLibrary
+Library    SeleniumLibrary
+
+
+*** Variables ***
+${STARBURST_CR_DEFAULT_NAMESPACE}=    redhat-starburst-operator
+${USERNAME_XP}=    xpath=//input[@id="username"]
+${STARBURST_ROUTE}=    web-ui
+
+
+*** Keywords ***
+Create Starburst Route
+    [Documentation]    Exposes starburst service for external connections
+    [Arguments]    ${name}
+    ${rc}=    Run And Return Rc    oc expose svc starburst --name ${name} -n ${STARBURST_CR_DEFAULT_NAMESPACE}
+    # ${rc}=    Run And Return Rc    oc create route edge ${name} --service=starburst --insecure-policy="Redirect" -n ${STARBURST_CR_DEFAULT_NAMESPACE}    # robocop: disable
```

Comment:
can this be removed?

Reply:
I'm keeping it because trying to understand the best way of creating the route. Some conversations with Staburst going on